### PR TITLE
Change the layout of the experience modal

### PR DESCRIPTION
**Pull request by colinhowe**
Resolves https://github.com/sparkletown/internal-sparkle-issues/issues/1513

Note: the delete button placement is temporary and will be getting moved to the list component in the near future.

New visuals:
image

### CODE REVIEW

File: src/components/organisms/TimingEventModal/TimingEventModal.scss [hunk 1]
```diff
@@ -13,6 +13,20 @@
     }
   }
 
+  &__duration_container {
+    input {
+      display: inline-block;
+      width: auto;
+      border: 0.1px solid var(--accent--over-20a);
```
Comment (goran-peoski-work) [inline src/components/organisms/TimingEventModal/TimingEventModal.scss:20]: we have 
```scss
$border-width--none: 0px;
$border-width--slim: 1px;
$border-width--bold: 2px;
```

should this `0.1px` be a thing? 
If not, is it a workaround? 
If so, maybe a comment above it why it isn't simply `none`

File: src/components/organisms/TimingEventModal/TimingEventModal.scss [hunk 2]
```diff
@@ -13,6 +13,20 @@
...
+      text-align: left;
+    }
+
+    label {
+      margin-left: 4px;
+      margin-right: 8px;
```
Comment (goran-peoski-work) [inline src/components/organisms/TimingEventModal/TimingEventModal.scss:26]:
```suggestion
      margin-left: $spacing--xs;
      margin-right: $spacing--sm;
```

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx [hunk 3]
```diff
@@ -246,26 +243,25 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
                 </div>
               </div>
 
-              <AdminSection title="Duration">
```
Comment (goran-peoski-work) [inline src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: Why remove `AdminSection` which is semantic and replace it with `<div  className="input-group">` which has a style from `global.scss`?

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx [hunk 4]
```diff
@@ -279,17 +275,13 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
...
-                  variant="primary"
-                  type="submit"
-                >
-                  {event ? "Update" : "Create"}
-                </ButtonNG>
-
+                {/*
+                  Note: this delete button is being moved to the experience
+                  list component in the near future
+                  */}
```
Comment (goran-peoski-work) [inline src/components/organisms/TimingEventModal/TimingEventModal.tsx:284]:
```suggestion
                { 
                  // @debt: move this delete button to the experience list component
                 }
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
Suggestions and questions

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx [hunk 5]
```diff
@@ -246,26 +243,25 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
...
-                      className="input-group__modal-input input-group__modal-input--indent"
-                      placeholder="minutes"
-                      ref={register}
-                    />
-                    minutes
-                  </label>
+              <div className="input-group">
+                <label>Duration</label>
+                <div className="TimingEventModal__duration_container">
+                  <input
```
Comment (alexey-petrov) [inline src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: We have `AdminInput` component which should fit fine and it's a general use component that we're reusing in admin v3

Comment (colinhowe) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: Had this discussion with Goran. I would *love* to use it. Doing so requires that I update a load of AV1 code due to field names not being what `AdminInput` expects. For now, we agreed to keep it as is.

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx (hunk 3 above)
Comment (colinhowe) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: It creates a H2 in the middle of form. That's not a convention in this form. I would rather update all the other divs than have this one being different. Given we're going to be redesigning all of this I'd rather not spend too much time on this kind of change.

File: src/components/organisms/TimingEventModal/TimingEventModal.scss (hunk 1 above)
Comment (colinhowe) [reply src/components/organisms/TimingEventModal/TimingEventModal.scss:20]: Existing styles. I'll update what's there to make it use the variables

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx (hunk 3 above)
Comment (goran-peoski-work) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: Not spending time === you don't change it in any way, even if there is h2 in the middle of the form. At least do not add `input-group`

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx (hunk 5 above)
Comment (goran-peoski-work) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: `<div className="input-group">` is still an issue for me. I'd simply leave the old section in place.

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx (hunk 3 above)
Comment (colinhowe) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: The h2 looks messy in the form with different font size etc.
image
The rest of the form uses `input-group`.

What's the issue here? Is this about trying to remove bootstrap?

Comment (goran-peoski-work) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: It's about trying to remove `input-group`. If you can localize the few style props to `TimingEventModal__input-group` even by just copying it from the global, that would be good enough I guess.

Comment (colinhowe) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:249]: I can do that. Cheers :)

File: src/components/organisms/TimingEventModal/TimingEventModal.scss [hunk 6]
```diff
@@ -23,12 +37,36 @@
     max-width: 700px;
   }
 
-  .input-group {
+  &__input-group {
```
Comment (goran-peoski-work) [inline src/components/organisms/TimingEventModal/TimingEventModal.scss:40]:
```suggestion
  // @debt this should be shorter, not having to compensate for `global.scss` interference. remove when no longer necessary
  &__input-group {
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM

I've added a `@debt` comment so we can clean it up in the future

**Pull request merged**